fe/fe-core/src/main/java/org/apache/doris/catalog/DatabaseIf.java

```diff
@@ -172,6 +172,19 @@ default T getTableOrMetaException(String tableName, List<TableIf.TableType> tabl
         return table;
     }
 
+    default T getTableOrMetaException(String tableName, TableIf.TableType... tableTypes)
+            throws MetaNotFoundException {
+        T table = getTableOrMetaException(tableName);
+        for (TableIf.TableType tableType : tableTypes) {
+            if (table.getType() == tableType) {
+                return table;
+            }
+        }
+        throw new MetaNotFoundException(
+                "Tye type of " + tableName + " doesn't match, expected data tables=" + tableTypes);
+
+    }
+
```

Comment:
The definition of this method looks similar as the above one (`default T getTableOrMetaException(String tableName, List<TableIf.TableType> tableTypes)
            throws MetaNotFoundException`). We should reuse the same logic.